pxtlib/browserutils.ts

```diff
@@ -126,6 +126,10 @@ namespace pxt.BrowserUtils {
         return hasNavigator() && /Epiphany/i.test(navigator.userAgent);
     }
 
+    export function isChromeOS(): boolean {
```

Comment:
Does this check pass for Android webview inside ChromeOS?

Reply:
switched to check for android